container.go

```diff
@@ -234,7 +234,7 @@ func (c *container) loadTask(ctx context.Context, ioAttach IOAttach) (Task, erro
 	if err != nil {
 		err = errdefs.FromGRPC(err)
 		if errdefs.IsNotFound(err) {
-			return nil, errors.Wrapf(err, "no running task found")
+			return nil, errdefs.ToGRPC(err)
```

Comment:
Why change this back to GRPC error. `cri-containerd` should use `errdefs` instead after switching to containerd client.